content/tutorial-regular-keys.md

```diff
@@ -0,0 +1,300 @@
+      "TransactionType": "SetRegularKey",
+      "TxnSignature": "304402204BCD5663F3A2BA02D2CE374439096EC6D27273522CD6E6E0BDBFB518730EAAE402200ECD02D8D2525D6FA4642613E71E395ECCEA01C42C35A668BF092A00EB649C26",
+      "hash": "AB73BBF7C99061678B59FB48D72CA0F5FC6DD2815B6736C6E9EB94439EC236CE"
+    }
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+The `sign` command response contains a `tx_blob` value, as shown above. The offline signing response contains a `signedTransaction` value. Both are signed binary representations (blobs) of the transaction.
+
+Next, use the `submit` command to transmit the transaction blob (`tx_blob` or `signedTransaction`) to the network.
+
+
+#### Submit Your Transaction
+
+Take the `signedTransaction` value from the offline signing response or the `tx_blob` value from the `sign` command response and submit it as the `tx_blob` value using the [`submit`](reference-rippled.html#submit) command.
+
+Here's an example `submit` request and response for a `SetRegularKey` type transaction.
+
+
+##### WebSocket Request:
+
+```
+{
+    "command": "submit",
+    "tx_blob": "1200052280000000240000000468400000000000000A73210384CA3C528F10C75F26E0917F001338BD3C9AA1A39B9FBD583DFFFD96CF2E2D7A7446304402204BCD5663F3A2BA02D2CE374439096EC6D27273522CD6E6E0BDBFB518730EAAE402200ECD02D8D2525D6FA4642613E71E395ECCEA01C42C35A668BF092A00EB649C268114830923439D307E642CED308FD91EF701A7BAA74788141620D685FB08D81A70D0B668749CF2E130EA7540"
+}
+```
+
+
+##### WebSocket Response:
+
+```
+{
+  "result": {
+    "engine_result": "tesSUCCESS",
+    "engine_result_code": 0,
+    "engine_result_message": "The transaction was applied. Only final in a validated ledger.",
+    "tx_blob": "1200052280000000240000000468400000000000000A73210384CA3C528F10C75F26E0917F001338BD3C9AA1A39B9FBD583DFFFD96CF2E2D7A7446304402204BCD5663F3A2BA02D2CE374439096EC6D27273522CD6E6E0BDBFB518730EAAE402200ECD02D8D2525D6FA4642613E71E395ECCEA01C42C35A668BF092A00EB649C268114830923439D307E642CED308FD91EF701A7BAA74788141620D685FB08D81A70D0B668749CF2E130EA7540",
+    "tx_json": {
+      "Account": "rUAi7pipxGpYfPNg3LtPcf2ApiS8aw9A93",
+      "Fee": "10",
+      "Flags": 2147483648,
+      "RegularKey": "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+      "Sequence": 4,
+      "SigningPubKey": "0384CA3C528F10C75F26E0917F001338BD3C9AA1A39B9FBD583DFFFD96CF2E2D7A",
+      "TransactionType": "SetRegularKey",
+      "TxnSignature": "304402204BCD5663F3A2BA02D2CE374439096EC6D27273522CD6E6E0BDBFB518730EAAE402200ECD02D8D2525D6FA4642613E71E395ECCEA01C42C35A668BF092A00EB649C26",
+      "hash": "AB73BBF7C99061678B59FB48D72CA0F5FC6DD2815B6736C6E9EB94439EC236CE"
+    }
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+Note that the response contains a `hash` of the transaction, which you can use to [look up the transaction's final outcome](reference-rippled.html#tx).
+
+
+### 3. Submit an `AccountSet` Transaction Using the Regular Key
```

Comment:
Consider changing this to something like "Verify Regular Key" since that's the purpose of the `AccountSet` transaction (as I understand it).